(Doc+) link videos for allocation and ilm (#116880)
elastic/elasticsearch: docs/reference/ilm/error-handling.asciidoc

until you resolve the underlying issues with the policy, index, or cluster.

> See this https://www.youtube.com/watch?v=VCIqkji3IwY[{ilm-init} health video] 

Reviewer: ```suggestion
See https://www.youtube.com/watch?v=VCIqkji3IwY[this video] 
```

> for example troubleshooting walkthrough of live issues and this https://www.youtube.com/watch?v=onrnnwjYWSQ[{ilm-init} history video]

Reviewer: ```suggestion
for a walkthrough of troubleshooting current {ilm-init} health issues, and https://www.youtube.com/watch?v=onrnnwjYWSQ[this video]
```

> for example troubleshooting walkthrough of historical issues.

Reviewer: ```suggestion
for a walkthrough of troubleshooting historical {ilm-init} issues.
```
